utils/rpms/pmdk.changelog

```diff
@@ -0,0 +1,218 @@
+* Fri Oct 31 2025  Tomasz Gromadzki <tomasz.gromadzki@intel.com> - 2.1.0-7
+- Restore the RPM changelog, which has not been available since version 2.1.0-4.
+
+* Fri Sep 12 2025  Jeff Olivier <jeffolivier@google.com> - 2.1.0-6
+- Fix leap package name
+
+* Thu Sep 11 2025  Jeff Olivier <jeffolivier@google.com> - 2.1.0-5
+- Fix pmdk package for leap
+- Expand the sds.at_create CTL to also cover pmemobj_open() (daos-stack/pmdk#5, DAOS-17449)
+  - Previously, this CTL affected only pmemobj_create().
+  - Now, it affects both pmemobj_create() and pmemobj_open().
+  - pmemobj_open() won't be able to open a pool with SDS enabled if the feature is currently
+  force-disabled.
+  - Conversely, pmemobj_open() does not issue a warning when attempting to open a pool with SDS disabled
+    while the feature is force-disabled.
+
+* Mon Aug 11 2025  Jeff Olivier <jeffolivier@google.com> - 2.1.0-4
```

Comment:
?

```suggestion
* Wed Sep 10 2025  Jeff Olivier <jeffolivier@google.com> - 2.1.0-4
```

Ref: https://github.com/daos-stack/daos/commit/3323a5d6fb2ea6b0181c75cacfa9528d17450e01

Reply:
For the first time, the fpm was introduced here:
https://github.com/daos-stack/daos/commit/3323a5d6fb2ea6b0181c75cacfa9528d17450e01

Reply:
Got it. Again you just copied from the DAOS spec's changelog. It is fine. Nvm.